app/src/features/sources/__tests__/SourceCard.test.tsx

```diff
@@ -16,6 +16,8 @@ jest.mock("services/api/api");
 const api = require("services/api/api");
 
 describe("SourceCard with source", () => {
+  beforeEach(cleanup);
```

Comment:
I think we don't need to use cleanup anymore now as it's being handled automatically : https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-cleanup